src/untangled/datomic/impl/components.clj

```diff
@@ -77,3 +79,7 @@
         (info "Deleting database" db-name url)
         (datomic/delete-database url)))
     (assoc this :connection nil)))
+
+(prefer-method io.aviso.exception/exception-dispatch clojure.lang.IPersistentMap clojure.lang.IDeref)
+(prefer-method print-method clojure.lang.IRecord clojure.lang.IDeref)
+(prefer-method clojure.pprint/simple-dispatch clojure.lang.IPersistentMap clojure.lang.IDeref)
```

Comment:
Those multimethods are all for printing, and when I was testing `deref` it was failing to find a preferred method because `IDeref` messes up the hierarchy. I figured adding them wouldn't hurt, but I hadn't thought about the case when `io.aviso.exception` isnt a dependency.